docs/docs/how-tos/auth/custom_auth.md

```diff
@@ -1,138 +1,147 @@
 # Add custom authentication
 
-!!! tip "Prerequisites"
+This guide shows how to add custom authentication to your LangGraph Platform application. This guide applies to both LangGraph Platform and self-hosted deployments. It does not apply to isolated usage of the LangGraph open source library in your own custom server.
 
-    This guide assumes familiarity with the following concepts:
+!!! note
 
-      *  [**Authentication & Access Control**](../../concepts/auth.md)
-      *  [**LangGraph Platform**](../../concepts/langgraph_platform.md)
-    
-    For a more guided walkthrough, see [**setting up custom authentication**](../../tutorials/auth/getting_started.md) tutorial.
+    Custom auth is supported for all **managed LangGraph Platform** deployments, as well as **Enterprise** self-hosted plans. It is not supported for **Lite** self-hosted plans.
 
-???+ note "Support by deployment type"
+## Add custom authentication to your deployment
 
-    Custom auth is supported for all deployments in the **managed LangGraph Platform**, as well as **Enterprise** self-hosted plans. It is not supported for **Lite** self-hosted plans.
+To leverage custom authentication and access user-level metadata in our deployments, you need to populate the `config["configurable"]["langgraph_auth_user"]` object through a custom authentication handler.
```

Comment:
The developer does NOT explicitly populate this. This variable is AUTOMATICALLY populated any time they set up custom auth.

Reply:
Updated the language here